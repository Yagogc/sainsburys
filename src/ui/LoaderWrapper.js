import styled from "styled-components";

export const LoaderWrapper = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  width: 100%;
  min-height: calc(100vh - 58px);
`;
