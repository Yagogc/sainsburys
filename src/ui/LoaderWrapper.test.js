import React from "react";
import renderer from "react-test-renderer";
import { LoaderWrapper } from "./LoaderWrapper";

test("UI - LoaderWrapper:", () => {
  const tree = renderer.create(<LoaderWrapper />).toJSON();
  expect(tree).toMatchSnapshot();
});
