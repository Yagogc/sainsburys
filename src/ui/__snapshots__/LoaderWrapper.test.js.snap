// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI - LoaderWrapper: 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
  min-height: calc(100vh - 58px);
}

<div
  className="c0"
/>
`;
